src/icons/index.ts

```diff
@@ -30,7 +30,6 @@ export const deleteIcon = POLARIS_ICONS.deleteIcon;
 export const disable = POLARIS_ICONS.disable;
 export const dispute = POLARIS_ICONS.dispute;
 export const duplicate = POLARIS_ICONS.duplicate;
-export const ellipsis = POLARIS_ICONS.ellipsis;
```

Comment:
For backwards compat, would this make sense?

```js
export const ellipsis = POLARIS_ICONS.horizontalDots;
```

Also, show a deprecation message when somebody is using ellipsis instead of horizontal dots?

And then in v4 we'd be able to completely remove it.

(unless I've missed something and this icon was already deprecated in polaris-react?)